server/balancer_worker.go

```diff
@@ -53,6 +55,7 @@ func newBalancerWorker(cluster *ClusterInfo, balancer Balancer, interval time.Du
 		balanceOperators: make(map[uint64]*BalanceOperator),
 		balanceCount:     defaultBalanceCount,
 		balancer:         balancer,
+		regionCache:      NewExpireRegionCache(interval, 2*interval),
```

Comment:
Yes, now it is only a filter, we have not used the cache value, but in further we may use it.
So i think it is better to do this after we decide to only keep it as a filter.